ethdb/privateapi/ethbackend.go

```diff
@@ -306,20 +306,22 @@ func (s *EthBackendServer) EngineGetBlobsBundleV1(ctx context.Context, in *remot
 
 // EngineNewPayload validates and possibly executes payload
 func (s *EthBackendServer) EngineNewPayload(ctx context.Context, req *types2.ExecutionPayload) (*remote.EnginePayloadStatus, error) {
+	fmt.Println("BC - override EngineNewPayload - Difficulty")
```

Comment:
Probably don't want this `fmt.Println` here? (use logger if we want to keep it?)

Reply:
Good catch! I forgot to delete this.

Reply:
Fixed.